targets/TARGET_STM/TARGET_STM32L4/device/stm32l4xx_hal_pcd.c

```diff
@@ -311,10 +310,10 @@ __weak void HAL_PCD_MspDeInit(PCD_HandleTypeDef *hpcd)
   */
 HAL_StatusTypeDef HAL_PCD_Start(PCD_HandleTypeDef *hpcd)
 { 
-  __HAL_LOCK(hpcd); 
+  // MBED PATCH __HAL_LOCK(hpcd); 
```

Comment:
Why was the lock removed?

Reply:
It has been removed in the F4 file so I just did the same...